### PR TITLE
Use ClimaArtifact data in twostr test

### PR DESCRIPTION
**Pull request by Espeer5**
## Purpose 

This PR removes the TwoStream test dependency on ClimaArtifacts by using the twoStream test data from ClimaArtifacts. Corresponding changes in ClimaArtifacts may be [here](https://github.com/CliMA/ClimaArtifacts/pull/62).

## To-do

 - [x] Pass TwoStream tests with data from ClimaArtifact

## Content

 - Adds a method to ClimaLand artifacts.jl to get path to twostr test data artifact
 - Use above method to pull twostr data artifact into twostr test

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

File: Artifacts.toml [hunk 1]
```diff
@@ -44,3 +44,10 @@ git-tree-sha1 = "043f9354b961fd3ef6ac4cf71d0c99930e177a92"
     [[bonan_richards_eqn.download]]
     sha256 = "50f1739dfd8193742488f249bd1ba8a157dfc7c1d8a27392ba7edbe9b4c0db03"
     url = "https://caltech.box.com/shared/static/4jcz9c1lp6rt750q28kx3qvihi8393tv.gz"
+
+[twostr_test]
+git-tree-sha1 = "b0986aa28e7970a9c080461f5e52fb125cda9c3d"
+
+    [[twostr_test.download]]
+    sha256 = "c972f3ff4e041a4443c6e970021f1084a1cc176c2aeeda453f3351ef7cffa60b"
+    url = "https://caltech.box.com/shared/static/zh7wb0ar8kwg4sio64id9e6otzcsfhad.gz"
```
Comment (Sbozzolo) [inline Artifacts.toml:53]: I still have to look at the PR in ClimaArtifacts, but if this artifact is only used for tests, we should make it a test artifact.
Namely, add an Artifacts.toml in the test folder and add this snippet in runtests
```
import Pkg, Artifacts
# Download test-only artifacts
#
# (Currently not natively supported by Julia)
artifacts_toml = joinpath(@__DIR__, "Artifacts.toml")
artifacts = Artifacts.select_downloadable_artifacts(artifacts_toml)
for name in keys(artifacts)
    Pkg.Artifacts.ensure_artifact_installed(name, artifacts[name], artifacts_toml)
end
```

**Review by kmdeck — COMMENTED** [review]
looks good to me! Thank you. I agree with Gabriele's comment, we could start a similar Artifacts.jl and toml in tests

File: test/standalone/Vegetation/test_two_stream.jl [hunk 2]
```diff
@@ -11,20 +11,14 @@ import ClimaComms
...
-    @__DIR__,
-    "PySellersTwoStream Data",
-    ArtifactFile[ArtifactFile(
-        url = "https://caltech.box.com/shared/static/e7angzdnw18tmf8gctse5flsrkjsrlhx.csv",
-        filename = "2_str_test_data.csv",
-    ),],
-)
-datapath = get_data_folder(data_file)
-data = joinpath(datapath, "2_str_test_data.csv")
+include(joinpath(climaland_dir, "test/Artifacts.jl"))
```
Comment (Sbozzolo) [inline test/standalone/Vegetation/test_two_stream.jl:16]:
```suggestion
include("../../Artifacts.jl"))
```

File: test/runtests.jl [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+# (Currently not natively supported by Julia)
+artifacts_toml = joinpath(@__DIR__, "Artifacts.toml")
+artifacts = Artifacts.select_downloadable_artifacts(artifacts_toml)
+for name in keys(artifacts)
+    Pkg.Artifacts.ensure_artifact_installed(
+        name,
+        artifacts[name],
+        artifacts_toml,
+    )
+end
```
Comment (Sbozzolo) [inline test/runtests.jl:17]: You can move this to the `Artifacts.jl` file

**Review by Sbozzolo — COMMENTED** [review]
You also have to add the Artifacts.toml file for `test`

**Pull request merged**